src/YUILoader.cc

```diff
@@ -156,6 +167,48 @@ void YUILoader::loadUI( bool withThreads )
     }
 }
 
+void YUILoader::loadRestAPIPlugin( const std::string & wantedGUI, bool withThreads )
+{
+    // Do not try to load if variable is not set
+    yuiMilestone () << "Requested to start http server to control UI." << std::endl;
+    if( pluginExists( YUIPlugin_RestAPI ) )
+    {
+        // Load underlying UI plugin, as test method inherits from it
+        // YUIPlugin_Test uses both libraries to be single point of entry
+        YUIPlugin uiPluginNC( YUIPlugin_NCurses );
+        YUIPlugin uiPluginQT( YUIPlugin_Qt );
+        YUIPlugin uiTestPlugin( YUIPlugin_RestAPI );
```

Comment:
IIUC libyui-rest-api is linked against both ncurses and Qt so we need to load both plugins. Am I correct?

Reply:
Yes, that's he main thing which I don't like, and I am not sure if it's worth to make 3 packages to have ncurses and qt parts isolated. As it will be library with single class in it. Disadvantage is obviously we load both libraries even not using it.